packages/fuel-indexer/src/service.rs

```diff
@@ -42,6 +42,10 @@ pub struct IndexerService {
 
     /// Killers used to stop the spawned indexers.
     killers: HashMap<String, Arc<AtomicBool>>,
+
+    /// For notifications that an indexer stopped execution after kill was
+    /// requested.
+    kill_confirms: HashMap<String, futures::channel::oneshot::Receiver<()>>,
```

Comment:
+1 for the slightly more clarified name of `kill_confirm_triggers` (or something similar)